lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -475,6 +540,40 @@ func TestCellularPartialGet(t *testing.T) {
 	}
 	tests.RunUnitTest(t, e, tc)
 
+	// happy path
+	tc = tests.Test{
+		Method:         "GET",
+		URL:            fmt.Sprintf("%s/%s/cellular/ngc/", testURLRoot, "n10"),
+		ParamNames:     []string{"network_id"},
+		ParamValues:    []string{"n10"},
+		Handler:        getNgc,
+		ExpectedStatus: 200,
+		ExpectedResult: tests.JSONMarshaler(&lteModels.NetworkNgcConfigs{
```

Comment:
Here and elsewhere: move the networkNgcConfigs to its own line

Reply:
done